recipe/meta.yaml

```diff
@@ -1,27 +1,19 @@
 {% set name = "mamba" %}
-{% set libmamba_version = "1.0.0" %}
-{% set libmambapy_version = "1.0.0" %}
-{% set mamba_version = "1.0.0" %}
-{% set release = "2022.11.01" %}
+{% set libmamba_version = "1.3.0" %}
+{% set libmambapy_version = "1.3.0" %}
+{% set mamba_version = "1.3.0" %}
+{% set release = "2023.02.03" %}
```

Comment:
1.3.1 with tag 2023.02.09 is out.
https://github.com/mamba-org/mamba/compare/2023.02.03...2023.02.09
You only need to update the version and sha256 to get it.